src/Idris/Pretty.idr

```diff
@@ -277,6 +281,7 @@ mutual
       go d (PSectionR _ x op) = parens (go startPrec x <++> pretty op)
       go d (PEq fc l r) = parenthesise (d > appPrec) $ go startPrec l <++> equals <++> go startPrec r
       go d (PBracketed _ tm) = parens (go startPrec tm)
+      go d (PString _ xs) = parenthesise (d > appPrec) $ hsep $ punctuate "++" (prettyString <$> xs)
```

Comment:
Is there a reason we are we not displaying it as an interpolated string?

Reply:
If I don't understand wrong, this is used to display error message from TTImp, and since resugar is not implemented for interpolated string, it's useless to show it in interploated form.